webapp/src/pages/ChatRoom/Messages.tsx

```diff
@@ -0,0 +1,93 @@
+import { useEffect, useState } from 'react';
+import { ChatBubble, ChatBubbleVariant } from '../../shared/components';
+import { User } from '../../shared/generated';
+import { useAuth } from '../../shared/hooks/UseAuth';
+import socket from '../../shared/socket';
+import { AVATAR_EP_URL, WILDCARD_AVATAR_URL } from '../../shared/urls';
+
+type MessageType = {
```

Comment:
this could come from the generated files, isn't it?

Reply:
It could be if we create at least one endpoint that returns this type of data and document it using Swagger.